assemblies/pentaho-cde-renderer/karaf/pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.pentaho.ctools</groupId>
+    <artifactId>pentaho-cde-renderer-assembly</artifactId>
+    <version>8.1.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>pentaho-cde-renderer-karaf-assembly</artifactId>
+
+  <packaging>karaf-assembly</packaging>
+
+  <properties>
+    <karaf.version>3.0.8</karaf.version>
+    <karaf-maven-plugin.version>4.1.4</karaf-maven-plugin.version>
```

Comment:
Any particular reason why you're using version 4.X of karaf-maven-plugin? 
I know that for feature generation we need to use the same version as the karaf distribution but I'm unsure of the impact here on the assembly of the karaf distro.